showcase/src/assets/translations/en.json

```diff
@@ -236,6 +236,20 @@
         "TITLE": "Stark Typography"
       }
     },
+    "HOMEPAGE": {
+      "MAIN_TITLE": "About Stark",
+      "DESCRIPTION_BEFORE_DETAIL": "<p>Stark is an Angular 5+ based front-end framework built on top of Angular Stark features... An awesome reusable build using Webpack, built-in support for state of the art front-end tech.</p> <p>Stark provides main building blocks for accelerating front-end development:</p>",
+      "DESCRIPTION_DETAIL": "<ul><li>a solid reusable build based on Webpack</li><li>a starter project inspired by Angular Starter by AngularClass and Google's Web Starterk Kit</li><li>core modules providing useful APIs</li><li>UI modules providing interesting UI components </li><li>a showcase application containing: <ul><li>our living style guide</li><li>a showcase of all components/services along with their API</li><li>our developer guide</li></ul></li></ul>",
+      "DESCRIPTION_AFTER_DETAIL": "<p>Stark modules are like LEGO blocks: add what you need, no less, no more. If you don't like our defaults then you may replace/override/ignore anything you fancy.</p> <a href=\"https://github.com/NationalBankBelgium\" target=\"_blank\" rel=\"noopener noreferrer\"><img class=\"github-link\" src=\"/assets/images/logo/github.png\"></a><p> And even better, the Stark repository is available on the National Bank of Belgium page on Github!</p>Come and have a look at our amazing open sources projects!</p>",
+      "NEWS_TITLE": "Stark News",
+      "NEWS_DESCRIPTION": "Want to find the hottest news about the Stark framework?",
+      "NEWS_CONTENT": "Have a look at the news page right <a target=\"_blank\" href=\"news\" rel=\"noopener noreferrer\">here!</a>",
+      "DOCUMENTATION_MAIN_TITLE": "Stark Documentation",
+      "DOCUMENTATION_MAIN_DESC": "<p>Want some more info about Stark? Take a look at our links just below!</p>",
+      "DOCUMENTATION_CORE_DESC": "<p>Stark Core, aka stark-core, is a module providing reusable APIs, such as a routing, logging, log shipping, etc.</p><p>You can find the latest API documentation for stark-core <a href=\"https://stark.nbb.be/api-docs/stark-core/latest\" target=\"_blank\" rel=\"noopener noreferrer\" >here</a>.</p>",
+      "DOCUMENTATION_UI_DESC": "<p>Stark UI provides reusable UI components, for example data table, message pane, ... but also themes.</p><p>The latest API doc for that module can be found <a href=\"https://stark.nbb.be/api-docs/stark-ui/latest\" target=\"_blank\" rel=\"noopener noreferrer\" >here</a>.</p>",
```

Comment:
At the end of this text, you can add something like: "Or check the API for previous releases" and add the link to all the docs: https://github.com/NationalBankBelgium/stark/tree/gh-pages/api-docs/stark-ui  (so we keep that link as before)